src/main/java/Duke.java

```diff
@@ -1,10 +1,177 @@
+import java.util.Locale;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public static final String LINE_PRINT = "____________________________________________________________\n";
```

Comment:
Nice use of static final(I can't really find anything wrong with your coding standards, salute)